Tabloid/Controllers/PostController.cs

```diff
@@ -24,6 +24,13 @@ public IActionResult Get()
             return Ok(_postRepo.GetAllPublishedPosts());
         }
 
+        [HttpGet("comments/{id}")]
+        public IActionResult GetWithComments(int id)
+        {
+            var post = _postRepo.GetPostByIdWithComments(id);
+            return Ok(post);
+        }
+        
```

Comment:
This is returning nothing if there are no comments on the post.

Reply:
Server-side is now returning what it should